apiary.apib

```diff
@@ -1688,6 +1688,103 @@ For example you should create a group for the Introduction to Programming course
+        + course_link (string, optional) - Link to the web page with information related to this Credential. Typically this is a course information or event information page.
+        + language (string, optional) - Language for the group. Currently we support "en", "es", "vi", "pt", "ja", "fr", "da", "nl", "fi", "no", "nb", "ro", "sv", "th", "tw", "zh", "tr", "ar", "he", "ms" and "de". Default language is en.
+        + attach_pdf (boolean, optional) - PDF of the Credential should be attached to email when recipient is informed. Default is false.
+        + blockchain (boolean, optional) - Enable or disable recording of these Credentials on a Blockchain.
+        + design_id (number, optional) - DEPRECATED. Certificate Design which the Group will use to display Credentials.
+        + certificate_design_id (number, optional) - Certificate Design which the Group will use to display Credentials.
+        + badge_design_id (number, optional) - Badge Design which the Group will use to display Credentials.
+        + primary_design_id (number, optional) - Primary Design which the Group will use to display Credentials.
+        + department_id (number, optional) - Department that the group belongs to. If not defined the group will be assigned to the organizations default department.
+        + meta_data (object, optional) - You can use this parameter to attach key-value strings to the object. Meta data is useful for storing additional, structured information on an object.
+        + learning_outcomes(array, optional) - Searchable keywords identifying skills credential-holders need to demonstrate (or have learned) in the process of earning this credential.
+        + generate_private_credential (boolean, optional) - The default value of `Credential.private` for the Group. The supported values are `true`, `false`, and `null`. When this is set to `true` or `false`, it will override `Department.generate_private_credential` and be used as the default of `Credential.private` when creating a new `Credential`. When this is set to `null`, it won't override `Department.generate_private_credential` which will be used for the default of `Credential.private` instead.
+        + auto_expiry (number, optional) - Set a date for when the credentials automatically expire after a certain amount of time has elapsed since the date of issue.
+        + signup_url (string, optional) - This provides a direct path for credential viewers to enroll in your course.
+        + signup_url_show (boolean, optional) - Show or hide the sign up url. The supported values are `true`, `false` and `null`.
+        + course_link_show (boolean, optional) - This links the credential data record to your information about the credential on your website. The supported values are `true`, `false` and `null`.
+        + organization_link_show (boolean, optional) - Show or hide the link to your website homepage. The supported values are `true`, `false` and `null`.
```

Comment:
Is this copied from create doc? asking so that I don't have to review the copy if it's the same as create.

Reply:
Yup, it's from create attributes section